apps/gitness/src/pages/pipeline-list.tsx

```diff
@@ -58,13 +75,35 @@ export default function PipelinesPage() {
             }
           ]
         }))}
+        LinkComponent={LinkComponent}
       />
     )
   }
 
+  //fetch projects api to get the list of projects
+  const { data: projects, isLoading } = useMembershipSpacesQuery({
+    queryParams: { page: 1, limit: 30, sort: 'identifier', order: 'asc' }
+  })
+
+  const projectsItem: Project[] =
+    // @ts-expect-error remove "@ts-expect-error" once type issue for "content" is resolved
+    projects?.content?.map(membership => ({
+      id: `${membership?.space?.id}`,
```

Comment:
unnecessary template literals
```suggestion
      id: membership?.space?.id,
```

Reply:
Since the api is return the number type of id but it is required by a string here. I will change the type in the playground.

Reply:
has been solved the type into the component in the playground